src/recensio/plone/profiles/default/actions.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="utf-8"?>
+<object xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+        meta_type="Plone Actions Tool"
+        name="portal_actions"
+>
+  <object meta_type="CMF Action Category"
+          name="portal_tabs"
+  >
+    <property name="title">Portal tabs</property>
+    <object meta_type="CMF Action"
+            name="rezensionen"
+            i18n:domain="recensio"
+    >
+      <property name="title"
+                i18n:translate=""
+      >label_nav_reviews</property>
+      <property name="description"
+                i18n:translate=""
+      />
+      <property name="url_expr">string:${portal/rezensionen/absolute_url}/rezensionen</property>
```

Comment:
I see this contains rezensionen a couple of times.
Does it make sense to change it to:
```
portal/rezensionen/rezensionen/absolute_url
```
?

Reply:
Shouldn't it rather be `string:${portal_url}/rezensionen/rezensionen` ?

Reply:
I was proposing that form because you had in your original PR `portal/rezensionen/absolute_url`.
Do whatever you like, but be consistent.
Or just don't be and ignore this nitpicky comment

Reply:
`${portal/rezensionen/absolute_url}` was from the actions.xml export I did on the testing database from https://recensio50.fe.rzob.gocept.net/
But actually I didn't understand whey it was constructing the URL like that.